docs/en/pid-issuing.rst

```diff
@@ -0,0 +1,511 @@
+    "code_challenge_method":"S256",
+    "authorization_details":[
+    {
+        "type":"openid_credential",
+        "format": "vc+sd-jwt",
+        "credential_definition": {
+            "type": ["eu.eudiw.pid.it"]
+        }
+    }
+    ],
+    "redirect_uri":"eudiw://start.wallet.it",
+    "client_assertion_type":"urn:ietf:params:oauth:client-assertion-type:jwt-bearer",
+    "client_assertion":$WalletInstanceAttestation$ 
+    }
+
+
+.. note::
+    **Federation Check:** PID Provider needs to check that the Wallet Provider is part of the federation and in addition it should verify Wallet Instance Attestation validity by checking its signature and the validity of the claims inside it. 
+
+**Step 7:** The PID Provider creates a new request URI representing this new authorization request and returns it to the Wallet Instance. 
+
+.. code-block:: 
+
+    HTTP/1.1 201 Created
+    Content-Type: application/json
+    Cache-Control: no-cache, no-store
+    
+    {
+        "request_uri":"urn:ietf:params:oauth:request_uri:bwc4JK-ESC0w8acc191e-Y1LTC2",
+        "expires_in": 60
+    }
+
+**Steps 8-9:** The Wallet Instance sends an authorization request to the PID Provider authorization endpoint.
+
+.. code-block:: 
+
+    GET /authorize?client_id=$thumprint-of-the-jwk-in-the-cnf-wallet-attestation$
+        &request_uri=urn%3Aietf%3Aparams%3Aoauth%3Arequest_uri%3Abwc4JK-ESC0w8acc191e-Y1LTC2 HTTP/1.1
+    Host: pid.it
+ 
+
+.. note::
+
+   **User Authentication and Consent:** The PID Provider performs the User authentication based on the requirements of eIDAS LoA High and asks the User for consent for the PID issuance. 
+
+**Steps 10-11:** The PID Provider sends an authorization code to the Wallet Instance. 
+
+.. note::
+
+    The Wallet Instance redirect URI is a universal or app link registered with the local operating system, so this latter will resolve it and pass the response to the Wallet Instance.
+ 
+.. code-block:: 
+
+    HTTP/1.1 302 Found
+    Location: eudiw://start.wallet.it?code=SplxlOBeZQQYbYS6WxSbIA&state=fyZiOL9Lf2CeKuNT2JzxiLRDink0uPcd
+ 
+**Steps 12-13:** The Wallet Instance creates a key for DPoP and a fresh DPoP proof for the token request to the PID Provider. DPoP provides a way to bind the access token to a certain sender (Wallet Instance) `[DPoP-draft16] <https://datatracker.ietf.org/doc/html/draft-ietf-oauth-dpop-16>`_. Thus, it mitigates the misuse of leaked or stolen access tokens at the credential endpoint of PID Provider as the attacker needs to present a valid DPoP proof.
+
+**Step 14:** The Wallet Instance sends a token request to the PID Provider's token endpoint using the authorization *code*, *code_verifier* and *DPoP proof*, and *Wallet Instance Attestation*. The *Wallet Instance Attestation* is used to perform the Wallet Instance authentication at the token endpoint as it is defined in [:rfc:`7521`]. 
+
+.. code-block::
+
+    POST /token HTTP/1.1
+    Host: pid.it
```

Comment:
please, use example.org in non-normative examples

Reply:
done